test/e2e/index.test.ts

```diff
@@ -250,7 +240,12 @@ describe('E2E Tests', () => {
         expect(event.data.fingerprint).to.include('abcd');
       });
 
-      it('Loaded via preload script with nodeIntegration disabled', async () => {
+      it('Loaded via preload script with nodeIntegration disabled', async function() {
+        if (majorVersion < 8) {
```

Comment:
I think we can probably leave this test enabled for all Electron versions as they are passing in the other PRs.